zc_install/includes/modules/pages/admin_setup/header_php.php

```diff
@@ -11,17 +11,12 @@
   $changedDir = (bool)$_POST['changedDir'];
   $adminDir = $_POST['adminDir'];
   $adminNewDir = $_POST['adminNewDir'];
-  if ($changedDir) {
-    $_POST['adminDir'] = $_POST['adminNewDir'];
-    $adminDir = $adminNewDir;

```

Comment:
Turns out removing these lines can result in the admin config file getting written to the wrong dir  ... or actually not getting written at all.